src/network/postgres_protocol_handler.cpp

```diff
@@ -343,13 +343,14 @@ ProcessResult PostgresProtocolHandler::ExecQueryMessage(InputPacket *pkt, const
         }
 
         if (param_values.size() > 0) {
-          statement_->GetPlanTree()->SetParameterValues(&param_values);
+          traffic_cop_->statement_->GetPlanTree()->SetParameterValues(&param_values);
         }
-        param_values_ = param_values;
+        traffic_cop_->param_values_ = param_values;
 
         auto status =
-                traffic_cop_->ExecuteStatement(statement_, param_values_, unnamed, nullptr, result_format_,
-                             results_, rows_affected_, error_message_, thread_id);
+                traffic_cop_->ExecuteStatement(traffic_cop_->statement_, traffic_cop_->param_values_, unnamed, nullptr,
```

Comment:
Similar here, if the statement, parameter values can be accessed within traffic_cop, we should change the API of ExecuteStatement and get rid of the arguments passing to the function